src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindOperationWithSortIntegrationTest.java

```diff
@@ -0,0 +1,168 @@
+package io.stargate.sgv2.jsonapi.api.v1;
+
+import static io.restassured.RestAssured.given;
+import static io.stargate.sgv2.common.IntegrationTestUtils.getAuthToken;
+import static net.javacrumbs.jsonunit.JsonMatchers.jsonEquals;
+import static org.hamcrest.Matchers.is;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.http.ContentType;
+import io.stargate.sgv2.api.common.config.constants.HttpConstants;
+import io.stargate.sgv2.jsonapi.testresource.DseTestResource;
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.TreeMap;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@QuarkusIntegrationTest
+@QuarkusTestResource(DseTestResource.class)
+public class FindOperationWithSortIntegrationTest extends CollectionResourceBaseIntegrationTest {
```

Comment:
missing cases:

* sort on non-existing column
* sort on multiple columns
* sort on numbers
* sort on boolean
* sort on null/non-null values

Reply:
Will do

Reply:
Also want to see:

1. sort with filter
2. sort with `findOne()` and `findOneAndUpdate()` (if in scope)

Reply:
@vkarpov15 

- Added sort with filter test case in #308 
- sorting for findOne and findOneAndUpdate sort not supported, is this needed?
- Projection is work in progress, will add a case once it's completed.

Reply:
Quick note: I do not think Mongo actually supports "sort" on `findOne()` (or `findOneAndUpdate()`).
Atlas shell allows it for find:

    db.arrays.find().sort({ "x":1 })

but it fails for

    db.arrays.findOne().sort({ "x":1 })

with error

```
Atlas atlas-v8vini-shard-0 [primary] examples> db.arrays.findOne().sort({ "x":-1 })
TypeError: db.arrays.findOne().sort is not a function
```

Reply:
@maheshrajamani Could you add a link here? One doc I saw listed driver-level "find" options which were not specific to "findOne()", but I may have missed something.

Reply:
https://www.mongodb.com/docs/manual/reference/method/db.collection.findOneAndUpdate/#:~:text=projection%3A%20%3Cdocument%3E%2C-,sort%3A%20%3Cdocument%3E%2C,-maxTimeMS%3A%20%3Cnumber%3E%2C

Reply:
@maheshrajamani Ah. Yes, you are right! Oddly, findOne -- https://www.mongodb.com/docs/manual/reference/method/db.collection.findOne/ -- has different layout for settings.
But even that may accept `sort` (possibly) as it refers to "findOptions" which lists a big set of options, without indicating that some are only applicable to "full" `find()`.

And playing with Atlas console a bit more, I think this does actually work:

   db.arrays.findOne({}, {}, { "sort" : { "x":-1 }})

So I think I agree now that "sort" is actually supported, even if `findOne()` documentation sort of hides it.